src/schema/2.0/schema.json

```diff
@@ -8,23 +8,39 @@
       "properties": {
         "type": {
           "type": "string",
-          "enum": ["DNS-TXT"]
+          "enum": ["DNS-TXT", "DID", "DNS-DID"]
         },
         "location": {
           "type": "string",
           "description": "Url of the website referencing to document store"
+        },
+        "key": {
+          "type": "string",
+          "description": "Public key associated"
         }
       },
-      "required": ["type", "location"],
-      "additionalProperties": false
+      "required": ["type"]
     },
     "issuer": {
       "type": "object",
       "properties": {
+        "id": {
+          "type": "string",
+          "description": "Issuer's id, DID can be used"
+        },
         "name": {
           "type": "string",
           "description": "Issuer's name"
         },
+        "revocation": {
```

Comment:
I think we said revocation is mandatory ? (but that will introduce a breaking change and will be v3).

Same as above => you wont be aware before using verify.

Reply:
This is a little hard to verify with the schema itself, the problem is we are using the child to verify the parent. One approach will be to define different types of `identity` like `identityWithRevocation` & `identityWithoutRevocation` and then inside them specify the internal structure, but i think that will lead to an even more confusing schema to read. I've went with the approach that this is made optional instead and we will leave to verify to settle it.